src/cr/cube/matrix/assembler.py

```diff
@@ -286,17 +303,22 @@ def pairwise_significance_p_vals(self, column_idx):
         Raises `ValueError if the cube-result does not include `overlaps`
         and `valid_overlaps` cube-measures.
         """
-        if self._cube_has_overlaps:
+        if self._cube.has_overlaps:
             # If overlaps are defined, calculate significance based on them
             return self._assemble_matrix(
-                self._measures.pairwise_p_vals_for_subvar(column_idx).blocks
+                self._measures.pairwise_p_vals_for_subvar.blocks
             )
-        base_column_idx = self._column_order[column_idx]
-        return self._assemble_matrix(
-            self._measures.pairwise_p_vals(base_column_idx).blocks
-        )
+        return self._assemble_matrix(self._measures.pairwise_p_vals.blocks)
+
+    @lazyproperty
+    def pairwise_significance_p_vals_scale_means(self):
+        """1D optional np.float64 ndarray of p_vals for scale means measure"""
+        return np.block(self._measures.pairwise_p_vals_scale_means.blocks)[
+            self._column_order
+        ]
```

Comment:
Isn't this a marginal? Why wouldn't we use the `._assemble_marginal()` method for this (or whatever it's called) instead of reimplementing it?

Reply:
i think that marginals has some logic like orientation and is_defined property. We don't have this for pairwise summaries